### ISSUE TITLE
Custom tags for library

### ISSUE THREAD

**Issue by suirobert** [Aidoku/Aidoku#14]
Tags to mangas like:
-Completed
-Planning
-Paused
To order our libraries and just show mangas with a certain tag or show them all

**Comment by AzSiAz** [comment]
Maybe not `tags` but multiple library ?

**Comment by Skittyblock** [comment]
Yeah, the plan is to do multiple library categories first. I'm leaving this issue open to remind me.
